src/index.html

```diff
@@ -17,11 +17,362 @@
+          class="card__img"
+          src="./images/imac.jpeg"
+          alt="card img"
+        />
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__code">Product code: 195434</div>
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__reviews">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <span class="card__price-inscript">Price:</span>
+          <span class="card__price-value">$2,199</span>
+        </div>
+        <button
+          class="button"
+          data-qa="hover"
+        >
+          buy
+        </button>
+      </div>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__img"
+          src="./images/imac.jpeg"
+          alt="card img"
+        />
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__code">Product code: 195434</div>
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__reviews">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <span class="card__price-inscript">Price:</span>
+          <span class="card__price-value">$2,199</span>
+        </div>
+        <button
+          class="button"
+          data-qa="hover"
+        >
+          buy
+        </button>
+      </div>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__img"
+          src="./images/imac.jpeg"
+          alt="card img"
+        />
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__code">Product code: 195434</div>
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__reviews">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <span class="card__price-inscript">Price:</span>
+          <span class="card__price-value">$2,199</span>
+        </div>
+        <button
+          class="button"
+          data-qa="hover"
```

Comment:
The `data-qa` attribute here is set to `hover`, which is inconsistent with the first button that uses `data-qa="card-hover"`. Ensure consistent naming for `data-qa` attributes across similar elements.